src/Plugin_Toggler.php

```diff
@@ -142,6 +165,42 @@ public function ajax_toggle_plugin_version() {
 		die();
 	}
 
+	/**
+	 * Retrieves the controls.
+	 *
+	 * @return string The HTML to use to render the controls.
+	 */
+	public function get_controls() {
+		$output  = '<h2>Plugin toggler</h2>';
+		$output .= '<form action="' . esc_url( admin_url( 'admin-post.php' ) ) . '" method="POST">';
+		$output .= wp_nonce_field( 'plugin_toggler', '_wpnonce', true, false );
+		$output .= '<input type="hidden" name="action" value="yoast_seo_plugin_toggler">';
+
+		$output .= '<input type="checkbox" ' . checked( $this->options['plugin_toggler'], true, false ) . ' name="plugin_toggler" id="plugin_toggler"/> <label for="plugin_toggler">Show plugin toggler.</label>';
```

Comment:
Also `empty()` check would be cleaner.

Reply:
instead of `checked`? That function is useful here...